jib-build-plan/src/main/java/com/google/cloud/tools/jib/api/buildplan/Platform.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.api.buildplan;
+
+import javax.annotation.Nonnull;
+
+public class Platform {
+  @Nonnull private String os;
+  @Nonnull private String architecture;
```

Comment:
We don't use `@Nonnull`. Everything not `@Nullable` is assumed non-null.

Also, always add `final` if field references never change.

Reply:
Intresting !! Done